packages/react/src/StateLabel/StateLabel.tsx

```diff
@@ -33,6 +33,19 @@ const octiconMap = {
   unavailable: AlertIcon,
 }
 
+const labelMap: Record<keyof typeof octiconMap, 'Issue' | 'Pull request' | ''> = {
+  issueOpened: 'Issue',
+  pullOpened: 'Pull request',
+  issueClosed: 'Issue',
+  issueClosedNotPlanned: 'Issue',
```

Comment:
non-blocking, but would we want to add "not planned" here? (i.e. "Issue, not planned")

Reply:
Great catch!